modules/programs/wezterm.nix

```diff
@@ -0,0 +1,600 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.programs.wezterm;
+
+  genericType = with types; oneOf [ str bool int float ];
+
+  boolToStr = boolVal: if boolVal then "true" else "false";
+
+  formatBindings = bindings:
```

Comment:
Would it, perhaps, be possible to use some Lua code formatter to provide the formatting instead of doing it manually? Nixpkgs seems to have a luaformatter package available.

Reply:
I've just pushed an update which does formatting through `luaformatter` in the best way that I could think of, if there's a better way to do it please let me know.